include/atheme/hooktypes.in

```diff
@@ -12,7 +12,7 @@
 # structure to indicate what should happen. They are not allowed to destroy
 # the object.
 #
-# The hooks channel_join, user_add, and user_nickchange, may destroy the
+# The hooks channel_join, chanuser_sync, user_add, and user_nickchange, may destroy the
```

Comment:
Do we want to bother re-wrapping this to 76, leave it as it is, or re-wrap to 116 like I do?

The days of the 80 column terminal are over, and GitHub renders at 120 too, so ...

Reply:
I've rewrapped to 76(-ish) for consistency with the surrounding comments for now.